modules/users/client/config/users.client.routes.js

```diff
@@ -364,23 +395,26 @@
       $stateProvider.
         state('profile.references', {
           url: '/references',
-          templateUrl: '/modules/users/views/profile/profile-view-references.client.view.html',
+          templateUrl: profileReferencesTemplateUrl,
           requiresAuth: true,
           noScrollingTop: true,
           abstract: true,
           data: {
             pageTitle: 'Profile references',
           },
         }).
+        /*
+        @TODO find out what is up with this, template did not exist
```

Comment:
I guess this missing template is because https://github.com/Trustroots/trustroots/pull/918 is still open?

Reply:
Yes, it was never present, and is unlikely to be in the future. The route is used, though.

I'd suggest to replace the `templateUrl: .....` with `template: ''`. Would that work for you here?

Reply:
Yup, that works, thanks!